CHANGELOG.md

```diff
@@ -1,3 +1,25 @@
+### citus v13.0.0 (January 17, 2025) ###
+
+* Adds support for PostgreSQL 17.2 (#7699, #7661)
+
+* Propagates MERGE ... WHEN NOT MATCHED BY SOURCE (#7807)
+
+* Adds JSON_TABLE() support in distributed queries (#7816)
+
+* Propagates MEMORY and SERIALIZE options of EXPLAIN (#7802)
+
+* Allows specifying an access method for distributed partitioned tables (#7818)
+
+* Adds support for exclusion constraints on distributed partitioned tables (#7733)
```

Comment:
We allow at most 80 characters in this changelog due to some limitations in Debian packaging infrastructure.

```suggestion
* Adds support for exclusion constraints on distributed partitioned tables
  (#7733)
```